### PR TITLE
Films view

### PR DESCRIPTION
**Pull request by gskerry**
Browser view updates... to compare/merge with @carolynyoo 's....
image

### CODE REVIEW

File: browser/js/films/films.html [hunk 1]
```diff
@@ -11,8 +9,30 @@
...
-	<a ng-href="/products/{{film._id}}">{{film.title}}</a>	
-  <p ng-repeat="category in film.categories">{{category.name}}</p>
+<!-- <div ng-repeat="film in films"> -->
+	<!-- <a ng-href="/products/{{film._id}}">{{film.title}}</a>	 -->
+  <!-- <p ng-repeat="category in film.categories">{{category.name}}</p> -->
   <!--   <p>{{film.categories.name}}</p> -->	
-</div>
\ No newline at end of file
+<!-- </div> -->
+
```
Comment (itsaliu90) [inline browser/js/films/films.html:17]: The name _browserfilm_ is a bit confusing to me, maybe we could name it something like _filmcard_?

**Comment by itsaliu90** [comment]
Honestly not really feeling the grey background, black border, and generally prefer some `border-radius` up in there. 

I think a grid-implementation would be cooler. But it's better than what we have now, so :cookie:

**Comment by carolynyoo** [comment]
@alexyuningliu this commit is obsolete since we decided to go w the grid view for 'all' page. but thank you to @gskerry for the buttons idea!! :)

**Comment by gskerry** [comment]
Sounds good gang. This was still WIP, but realized @carolynyoo was working through same, so pushed. Worthwhile pieces incorporated into merged, so will close.

**Pull request closed without merging**